letsencrypt/cli.py

```diff
@@ -127,9 +127,9 @@ def _determine_account(args, config):
         elif len(accounts) == 1:
             acc = accounts[0]
         else:  # no account registered yet
-            if args.email is None:
+            if args.email is None and not args.allow_unsafe_registration:
```

Comment:
I think this is fine, though perhaps the flag should have a more accurate name, such as `--register-unsafely-without-email`.  The purpose of the flag is either use by folks who've decided they really don't want to provide an email, or automation in environments contact-based recovery is not in use.